hail/generate-build-info.sh

```diff
@@ -2,16 +2,25 @@
 
 set -e
 
+REVISION=$(git rev-parse HEAD)
+DATE=$(date -u +%Y-%m-%dT%H:%M:%SZ)
+DATE=$(git rev-parse --abbrev-ref HEAD)
+URL=$(git config --get remote.origin.url)
+SPARK_VERSION=$1
+HAIL_VERSION=$2
+
 echo_build_properties() {
+  echo "[Build Metadata]"
   echo user=$USER
-  echo revision=$(git rev-parse HEAD)
-  echo branch=$(git rev-parse --abbrev-ref HEAD)
-  echo date=$(date -u +%Y-%m-%dT%H:%M:%SZ)
-  echo url=$(git config --get remote.origin.url)
-  echo sparkVersion=$1
-  echo hailVersion=$2
+  echo revision=$REVISION
+  echo branch=$BRANCH
+  echo date=$DATE
+  echo url=$URL
+  echo sparkVersion=$SPARK_VERSION
+  echo hailVersion=$HAIL_VERSION
 }
 
 mkdir -p src/main/resources/
 
 echo_build_properties $1 $2 > "src/main/resources/build-info.properties"
```

Comment:
Also, this echo function business is a bit unusual.  I'd just use the

```
cat <<DONE
blah blah blah
DONE
```

idiom.  But this is really an unrelated change.

Reply:
yeah, that's what I would have done too